utils/kata-manager.sh

```diff
@@ -146,7 +146,23 @@ github_get_release_file_url()
 			;;
 
 		*containerd*)
-			[ "$arch" = "x86_64" ] && arch="amd64"
+			case "${arch}" in
+				x86_64*)
+					arch=amd64
+					;;
+				aarch64*)
+					arch=arm64
+					;;
+				s390x*)
+					arch=s390x
+					;;
+				ppc64le*)
+					arch=ppc64le
+					;;
+				*)
+					die "Unsupported arch. Must be x86_64, arm64, s390x, and ppc64le." ;;
```

Comment:
Nit: It should be "or" not "and":

```suggestion
				*)
					die "Unsupported arch. Must be x86_64, arm64, s390x, or ppc64le." ;;
```